### PR TITLE
Cloudanix Workspace added

### PR DESCRIPTION
**Pull request by ankitwagh617**

### CODE REVIEW

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
Can you make changes in build queries logic also?

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Comment by ganeshjagdale9545** [comment]
@mpurusottamc we need to test this before deployment

File: cartography/graph/cleanupbuilder.py [hunk 1]
```diff
@@ -104,13 +104,33 @@ def _build_cleanup_node_and_rel_queries(
...
+            <-[:OWNER]-(:AWSOrganization{id: $ORGANIZATION_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})
+            $selected_rel_clause
+            $delete_action_clause
+            """,
+        )
+    elif node_schema.sub_resource_relationship.target_node_label == "GitHubOrganization":
+        query_template = Template(
+            """
+            MATCH (n:$node_label)$sub_resource_link(:$sub_resource_label{$match_sub_res_clause})
+            <-[:OWNER]-(:GitHubOrganization{id: $ORGANIZATION_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})
```
Comment (ganeshjagdale9545) [inline cartography/graph/cleanupbuilder.py:120]: `GitHubOrganization` is sub_resource_label and it already in model, we need to add only 
`<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})` here

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
one minor change needed

File: cartography/graph/cleanupbuilder.py (hunk 1 above)
Comment (ankitwagh617) [reply cartography/graph/cleanupbuilder.py]: Done

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
lgtm

**Pull request merged**